OWNERS_ALIASES

```diff
@@ -25,10 +24,10 @@ aliases:
     - codificat
     - ashleyschuett
     - zcahana
+    - dhiller # 2024-03-15
```

Comment:
fwiw, the date comment is not really needed as this would be tracked down by git anyway.

Reply:
True. Not sure what my intent behind that was. It think I saw it in other OWNERS files and found the idea nice. I can remove it, if it should comply with general style.

Reply:
I actually saw it in the [owners docs](https://www.kubernetes.dev/docs/guide/owners/#emeritus)

Reply:
Ah, ok. It was just an observation on my side. Thanks for clarifying though.